haskell/private/dependencies.bzl

```diff
@@ -21,6 +21,17 @@ def _is_shared_library(f):
     """
     return f.extension in ["so", "dylib"] or f.basename.find(".so.") != -1
 
+def _is_static_library(f):
+    """Check if the given File is a static library.
+
+    Args:
+      f: The File to check.
+
+    Returns:
+      Bool: True if the given file `f` is a static library, False otherwise.
+    """
+    return f.extension in ["a"]
```

Comment:
Isn’t the information how a target has been linked propagated somehow?

Reply:
I don't know, I'll investigate.

Reply:
Not I'm aware of.